Update API following v2/v3 unification

harveytims opened the pull request:
The latest OAS have been converted into a Postman collection using the open source converter.

This has created a collection that mirrors what is documented upstream.

However, some of the formatting of the descriptions is off due to how the converter interpreted the Stoplight flavoured markdown. @MatthewJWoolley wondering whether you can help with this?

harveytims committed:
Update API following v2/v3 unification

MatthewJWoolley commented:
Reviewed the formatting issues and the only one I've spotted are Notes importing with `theme:warning` etc. in certain places. Will merge with these left in

MatthewJWoolleyViva committed:
Update environment and adjust legacy authorisation

- Add apiKey and change access_token to bearerToken
- Add tests to the legacy authorisation folder so that it handles bearerTokens gracefully
